x-pack/plugins/alerting/server/task_runner/create_execution_handler.test.ts

```diff
@@ -75,6 +77,37 @@ test('calls actionsPlugin.execute per selected action', async () => {
           },
         ]
     `);
+
+  const eventLogger = createExecutionHandlerParams.eventLogger;
+  expect(eventLogger.logEvent).toHaveBeenCalledTimes(1);
+  expect(eventLogger.logEvent.mock.calls).toMatchInlineSnapshot(`
+    Array [
+      Array [
+        Object {
+          "event": Object {
+            "action": "execute-action",
+          },
+          "kibana": Object {
+            "alerting": Object {
+              "instance_id": "2",
+            },
+            "namespace": "default",
+            "saved_objects": Array [
+              Object {
+                "id": "1",
+                "type": "alert",
+              },
+              Object {
+                "id": "1",
+                "type": "action",
+              },
+            ],
+          },
+          "message": "alert: test:1: name-of-alert instanceId: 2 scheduled actionGroup: default action: test:1",
```

Comment:
Can we perhaps use quotes around the IDs just to make this a bit more readable?
As we'll likely be displaying this in the UI at some point, it might be worth figuring out how to make these clearer.

Reply:
Ya, I'll add some quotes - I think I had them in for a while, but removed them during some clean up.

We'll have to have a discussion about these `message` properties.  I'm currently thinking they're useless.  Messages like this are supposed to be for humans to help search through the bits, but the current use cases don't include stories like that.  We should probably have all the data they need as fields in the docs.  Or perhaps easier, feed all the fields into a text field via `copy_to`, which gives us the text search capability.

Note today, the message does include info like alert name, type, that is not yet in the actual event doc.  Denormalizing those bits is a bullet point in the umbrella issue https://github.com/elastic/kibana/issues/62221